src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -7,7 +7,7 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <RootNamespace>System.Xml</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <NoWarn>$(NoWarn),649</NoWarn>
+    <NoWarn>$(NoWarn),649,414</NoWarn>
```

Comment:
This warning was previously suppressed in System.Xml.XmlSerializer because of the error in some configuration, but you are right, there is no good reason to do this for this type of error. I'll remove this in a next commit.